cmd/did-method-cli/createconfigcmd/createconfig.go

```diff
@@ -79,10 +78,11 @@ type memberData struct {
 	Policy models.StakeholderSettings `json:"policy"`
 	// Endpoints is a list of sidetree endpoints owned by this stakeholder organization
 	Endpoints []string `json:"endpoints"`
-	// PrivateKeyJwk is privatekey jwk
-	PrivateKeyJwk json.RawMessage `json:"privateKeyJwk,omitempty"`
+	// PrivateKeyJwk is privatekey jwk file
+	PrivateKeyJwkPath string `json:"privateKeyJwkPath,omitempty"`
```

Comment:
this seems odd - why are you requiring a path?

Reply:
address comment from pervious PR https://github.com/trustbloc/trustbloc-did-method/pull/147#discussion_r429929811

Reply:
also - seems odd that the private key is part of this structure at all.

Reply:
we need to feed it to cli